src/tests/ftest/util/cont_security_test_base.py

```diff
@@ -0,0 +1,319 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+
+import os
+import pwd
+import grp
+import re
+from apricot import TestWithServers
+from avocado import fail_on
+
+import dmg_utils
+from daos_utils import DaosCommand
+from command_utils import CommandFailure
+import security_test_base as secTestBase
+#TODO: Uncomment 'general_utils' once the PR1484 has landed.
```

Comment:
(lint) Uncomment 'general_utils' once the PR1484 has landed.

Reply:
Forgot to remove this TODO comment. Done